workshops/personal_website/README.md

```diff
@@ -0,0 +1,575 @@
+  **`200px`**
+
+> **`px`** stands for **pixels**. **Pixels** are the tiny dots that make up a
+> computer screen. By setting the width to **200px**, we are saying we want
+> every image on the page to span the length of 200 pixels (which is about 2
+> inches on a screen). If you want to learn more about pixels, you can Google
+> ["What is a pixel?"](https://www.google.com/#q=what+is+a+pixel)
+
+### 4) Centering the Body
+
+Then to center everything on the page
+
+1. Type the below CSS code underneath our previous CSS code. _Remember, don't
+   copy and paste._
+
+   ```css
+   body {
+       text-align: center;
+   }
+   ```
+3. Save the file by clicking **"File"** → **"Save"** (or use the shortcut
+   **CTRL + S / Command + S**)
+
+> ![](img/css_text_align_center.gif)
+
+Now everything should be centered on the page.
+
+#### Understanding More Deeply
+
+If we were to translate the code into English, this
+
+```css
+body {
+    text-align: center;
+}
+```
+
+would  say:
+
+- **Select** every **`body` tag** on this HTML page and set the **alignment** of
+  the **text** within the body tag to be **centered**
+
+This centers everything on our page because all of the content in our HTML file
+is written inside the body tag.
+
+And because everything is in the **body** tag, everything is centered.
+
+> You can Google ["css text align"]
+  (https://www.google.com/?gws_rd=ssl#q=css+text+align) to see what other
+  alignment options are available besides **`center`**
+
+### 5) Changing the Font
+
+Finally let's change the font of the page to **`Arial`**.
+
+Add another **CSS attribute** to the **`body` selector** so that your CSS code
+for your **`body`** looks like this:
+
+  ```css
+  body {
+      text-align: center;
+      font-family: Arial;
+  }
+  ```
+
+> ![](img/css_font_family_arial.gif)
+
+> Not all computers have the same fonts on them. To figure out what fonts most
+> computers have on them, you can Google
+> ["web safe fonts"](https://www.google.com/#q=web+safe+fonts)
+
+### 6) Celebrate!
+
+![](img/celebrate_harry_potter.gif)
+
+W0000t! You've successfully added the CSS to this web page!
+
+Now to share your creation with the world.
+
+## Part V: Deployment
+
+Right now we can only see our website on our own computer. Let's get a link that
+we can share with anyone on the internet!
+
+### 1) Hosting on Cloud9
+
+In the **live preview** of the website:
+
+1. Click the button that looks like this
+
+   > ![](img/live_preview_button.png)
+
+2. Then in the new window that pops up, copy the URL of the address bar
+3. You can send this URL to anyone else in the world and they will be
+   able to view your website!
+
+**But there's a problem:**
+
+This URL won't work indefinitely, it will stop working when you don't use your
+Cloud9 for a couple of days. We want something that other people will be able to
+open even if it's a year from now.
+
+We can use **GitHub Pages** to **host** our website instead!
+
+#### 1) Pushing to GitHub Pages
```

Comment:
:+1: feels odd to me too, but if they make the workshop easier to follow,
I'm down

On Thu, Jan 21, 2016 at 2:45 PM, Max Wofford <notifications@github.com>
wrote:

> In workshops/personal_website/README.md
> <https://github.com/hackclub/hackclub/pull/652#discussion_r50477187>:
>
> > +
> > +   > ![](img/live_preview_button.png)
> > +
> > +2. Then in the new window that pops up, copy the URL of the address bar
> > +3. You can send this URL to anyone else in the world and they will be
> > +   able to view your website!
> > +
> > +**But there's a problem:**
> > +
> > +This URL won't work indefinitely, it will stop working when you don't use your
> > +Cloud9 for a couple of days. We want something that other people will be able to
> > +open even if it's a year from now.
> > +
> > +We can use **GitHub Pages** to **host** our website instead!
> > +
> > +#### 1) Pushing to GitHub Pages
>
> Do you want numbering in headers? It seems odd. Maybe that's just me.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/hackclub/pull/652/files#r50477187>.
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>